site/docs/build-ref.html

```diff
@@ -348,10 +348,11 @@ <h4>Package names, <code>//<b>package-name</b>:...</code></h4>
 </p>
 
 <p>
-  Although Bazel allows a package at the build root (e.g. <code>//:foo</code>), this
-  is not advised and projects should attempt to use more descriptively named
-  packages.
+  Although Bazel allows targets to be declared in the package at the build root
+  (e.g. <code>//:foo</code>), you are encouraged to leave that package empty:
+  it is better if every target is in a package having a descriptive name.
```

Comment:
I think this clarification is fair but still wordy in this draft. So without changing the substance I'd suggest something like:

"Although Bazel supports targets in the workspace's root package (e.g. <code>//:foo</code>), it's best to leave that package empty so all meaningful packages have descriptive names".

Also note [Bazel's doc style guide](https://bazel.build/basics/docs-style.html) discourages Latin abbreviations ([parent doc reference](https://developers.google.com/style/abbreviations)). Maybe "you" is still better than "it's" as I suggest ([reference](https://bazel.build/basics/docs-style.html#subject)). Also try to use active voice throughout ([reference](https://bazel.build/basics/docs-style.html#tense), my suggestion isn't perfect with that).

Reply:
Please take my pedantic grammar comments as a sign that I support the main point of this change.

Reply:
Love your suggestion, “omit needless words” can be challenging.  Next time use the "insert a suggestion" button and it will save substantial labor :-)

Reply:
Well I just learned about a new GitHub feature. :)

I'd like to give @aiuto another opportunity to respond. If there's no more update after a day or two, please re-ping and I'll get this approved and merged.

Reply:
@aiuto it looks like this PR is waiting on you.